packages/nimble-components/src/table-column/enum-base/models/column-validator.ts

```diff
@@ -0,0 +1,40 @@
+import { Mapping } from '../../../mapping/base';
+import type { MappingKeyType } from '../types';
+
+/**
+ * Helper to share logic between the enum-text and icon column validators
+ */
+export class TableColumnEnumValidationHelper {
+    public static invalidMappingKeyValueForType(
+        keys: (string | boolean | number | undefined)[],
```

Comment:
I've seen `string | boolean | number` used in a few different places. Should it be defined as a type somewhere? Maybe called something like `MappingValueType`.

Reply:
Created a type called `MappingKeyValue`, as that aligns with our `TableFieldValue` type.